pkgs/games/minetest/default.nix

```diff
@@ -125,14 +125,9 @@ stdenv.mkDerivation (finalAttrs: {
       prometheus-cpp
     ];
 
-  postPatch =
-    ''
-      substituteInPlace src/filesys.cpp \
-        --replace-fail "/bin/rm" "${coreutils}/bin/rm"
```

Comment:
Looks like they changed it to relative `"rm"` (https://github.com/minetest/minetest/pull/15238). I think we still want to patch it

Reply:
Good catch. Do you think `"rm"` is a good enough target for substitute?

Reply:
I'd write

```
--replace-fail 'argv = {
			"rm"' 'argv = { "${coreutils}/bin/rm"'
```

or if that doesn't work I'd use a patch instead